<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bTestBoxDialog</class>
 <widget class="QDialog" name="bTestBoxDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="languageCoverageLayout">
     <item>
      <layout class="QHBoxLayout" name="languageLayout">
       <item>
        <widget class="QLabel" name="languageLabel">
         <property name="text">
          <string>Language:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="languageComboBox">
         <item>
          <property name="text">
           <string>C</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ADA</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="translationProfileLayout">
       <item>
        <widget class="QLabel" name="translationProfileLabel">
         <property name="text">
          <string>Translation Profile:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="translationProfileComboBox">
         <item>
          <property name="text">
           <string>C9X</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PROJECT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>LIGHT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>01</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="coverageLabel">
         <property name="text">
          <string>Coverage Criteria:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="coverageComboBox">
         <item>
          <property name="text">
           <string>Code Coverage</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Path Coverage</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Branch Coverage</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Clause Coverage</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="operationsToTestScrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="operationsScrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>288</width>
        <height>88</height>
       </rect>
      </property>
      <widget class="QLabel" name="operationsLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Operations To Test:</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="allOperationsCheckBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>All Operations</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="testProjectNameLayout">
         <item>
          <widget class="QLabel" name="testProjectNameLabel">
           <property name="text">
            <string>Test Project Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="testProjectNameLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="optionalArgumentsLayout">
         <item>
          <widget class="QLabel" name="optionalArgumentsLabel">
           <property name="text">
            <string>Optional Arguments:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="optionalArgumentsLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="compilerLayout">
       <item>
        <widget class="QLabel" name="compilerLabel">
         <property name="text">
          <string>Compiler:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="compilerLineEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="notImplementedLabel">
     <property name="text">
      <string>Not Implemented Yet.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomButtonsLayout">
     <item>
      <widget class="QPushButton" name="advancedButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Advanced</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
